src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -196,7 +208,11 @@ public DRSAccessURL getAccessUrlForObjectId(
 
     DrsId drsId = drsIdService.fromObjectId(objectId);
     Snapshot snapshot = snapshotService.retrieve(UUID.fromString(drsId.getSnapshotId()));
-    GoogleProjectResource projectResource = snapshot.getProjectResource();
+
+    BillingProfileModel billingProfileModel =
+        profileService.getProfileById(snapshot.getProfileId(), authUser);
+
+    CloudPlatformWrapper wrapper = CloudPlatformWrapper.of(billingProfileModel.getCloudPlatform());
```

Comment:
nit, but I've been naming this variable platform:
```suggestion
    CloudPlatformWrapper platform = CloudPlatformWrapper.of(billingProfileModel.getCloudPlatform());
```